src/esp/gfx/WindowlessContext.cpp

```diff
@@ -60,16 +60,6 @@ const int MAX_DEVICES = 128;
     CHECK(err == EGL_SUCCESS) << "EGL error:" << err; \
   } while (0)
 
-bool isNvidiaGpuReadable(int device) {
-  const std::string dev = "/dev/nvidia" + std::to_string(device);
-  const int retval = open(dev.c_str(), O_RDONLY);
-  if (retval == -1) {
-    return false;
-  }
-  close(retval);
-  return true;
-}
```

Comment:
Great, with this removed the implementation moves closer to what I have in `WindowlessEglApplication` :D At this point I think the vanilla one is only missing a `--magnum-cuda-device N` counterpart to `--magnum-device N` and it could fully replace this code.